src/adr/0069-crowdsec-IP-filtering.md

```diff
@@ -20,19 +27,24 @@ A quick introduction about Crowdsec:
 CrowdSec Security Engine is an open-source and lightweight software that allows you to detect peers with malevolent behaviors and block them from accessing your systems at various levels (infrastructural, system, applicative).
 
 To achieve this, the Security Engine reads logs from different sources (files, streams ...) to parse, normalize and enrich them before matching them to threats patterns called scenarios. 
+
 ```
 
-## Decision 
+CrowdSec ships by default with scenarios (brute force, port scan, web scan, etc.) adapted for most contexts, but you can easily extend it by picking more of them from the [HUB](https://app.crowdsec.net/hub/collections). It is also easy to adapt an existing one or create one yourself.
 
 Set up a new maven project dedicated to Crowdsec extension. This allows to be embedded in a James server as a soft dependency
 using the external-jar loading mechanism. With this way, the extension could be dropped in one's James installation, and not a runtime dependency.
 
-To connect to Crowdsec, we use https protocol with reactor http client. 
+James connects to Crowdsec via https protocol with reactor http client. 
 
 ## Consequences
 
+James will be more secure by blocking threats at various levels (applicative, system, infrastructural). 
```

Comment:
- We can document and share Crowdsec scenari thus allowing users to benefit from each other decisions.